powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/ValidateTokenRequest.java

```diff
@@ -29,12 +30,21 @@
 @Data
 public class ValidateTokenRequest {
 
+    @NotNull
     private String tokenId;
+
     @ToString.Exclude
+    @NotNull
     private String tokenDigest;
+
+    @NotNull
     @ToString.Exclude
     private String nonce;
+
+    @NotNull
```

Comment:
First of all, it is propagated into swagger. Last but not least, it will be helpful if we switch to bean validation.